web/packages/teleport/src/components/ReAuthenticate/useReAuthenticate.ts

```diff
@@ -51,13 +68,18 @@ export default function useReAuthenticate({ onAuthenticated, onClose }: Props) {
     submitWithWebauthn,
     auth2faType: cfg.getAuth2faType(),
     preferredMfaType: cfg.getPreferredMfaType(),
+    dialogText,
```

Comment:
I'd pass this directly from `Container`'s props to `ReAuthenticate` without it going through the hook, but I don't want to barge in with new practices in an arbitrary PR.

Reply:
For the time being I think the current approach keeps our type cleaner, we will be shifting away from using `Container` with hooks in the future anyways.